invenio_accounts_rest/views.py

```diff
@@ -41,15 +41,20 @@
 from werkzeug.local import LocalProxy
 
 from invenio_accounts_rest.errors import MaxResultWindowRESTError
-from invenio_accounts_rest.loaders import default_json_loader_with_profile, \
-    default_json_loader_without_profile, \
-    default_json_patch_loader_with_profile, \
-    default_json_patch_loader_without_profile
+from invenio_accounts_rest.loaders import (
```

Comment:
Normally we don't use this style of import. isort ought to have caught this and should be able to auto format it so perhaps there's something wrong with the isort configuration.